### DISCUSSION TITLE
A Bill Entitled | JA-Legislation-Demo

### DISCUSSION THREAD

**Discussion by giscus** [Omnifk-DAO/JA-Legislation-Demo#8] [NIDS]
# A Bill Entitled | JA-Legislation-Demo

https://omnifk-dao.github.io/JA-Legislation-Demo/NIDS/1/

**Comment by Bertitude** [comment]
Turns out if I rename a page it gets a new topic. Interesting.

**Reply by Bertitude** [reply]
I am recommending this amendment
```
This act should be named Jamaica Identification and Registration Act, 2021
```
